TestModel/RoleTest.java

```diff
@@ -0,0 +1,34 @@
+import Model.Common.Role;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @autor: JuanaRodriguez
+ */
+public class RoleTest {
+    @Test
+    public void VerifyThatANewRoleWasAdd() {
+        Role role = new Role();
+        role.setRoleName("Administrator2");
+        role.save();
+        Role recoverRole = new Role();
+        recoverRole.getRoleById(role.getRoleId());
+        Assert.assertEquals("Administrator2", recoverRole.getRoleName());
+        //Assert.assertNotNull(role.getRoleById(role.getRoleId()));
+
+    }
+
+    @Test
+    public void VerifyThatARoleWasUpdated() {
+        Role role = new Role();
+        role.setRoleName("Gerent Edited1");
+        role.update(1);
+        Assert.assertEquals("Gerent Edited1", role.getRoleName());
```

Comment:
How you are updating the role here, i dont see any property that is change to verify that is updated